hibernate-core/src/main/java/org/hibernate/hql/internal/ast/tree/ImpliedFromElement.java

```diff
@@ -23,6 +23,21 @@
 	 */
 	private boolean inProjectionList;
 
+	/**
+	 * Here to add debug breakpoints
+	 */
+	@SuppressWarnings("unused")
+	public ImpliedFromElement() {
+		super();
+	}
+
+	/**
+	 * Here to add debug breakpoints
+	 */
+	public ImpliedFromElement(FromClause fromClause, FromElement origin, String alias) {
+		super( fromClause, origin, alias );
+	}
+
```

Comment:
another minor, the `ImpliedFromElement(FromClause fromClause, FromElement origin, String alias) ` constructor is never used, may be it can be deleted